lib/ImportFromC/HeaderToModel.cpp

```diff
@@ -485,15 +505,19 @@ DeclVisitor::getModelTypeForClangType(const QualType &QT) {
       else if (auto Rw = makeTypeByNameOrID<model::RawFunctionDefinition>(Name))
         R = std::move(Rw);
       else
-        Errors.emplace_back("import-from-c: Couldn't find function type in the "
-                            "model.\n");
+        Errors.emplace_back("import-from-c: Unknown typedef: '" + Name.str()
+                            + "'. Is it present in the model?\n");
```

Comment:
And what if not? See other places too.

Do you know that it's not present in the model? If so, just state it instead of asking questions.

If you ask questions you should also provide further directions depending on the answer to the questions, and the error message becomes a walkthrough.

I'd drop the question from the error message.

Reply:
No, that was just a cheeky way of saying "hey, check your model, you probably forgot to add it".

But yeah, I'm fine with just `Unknown typedef: '" + Name.str() + "'.`

Reply:
Done.